recipe-client-addon/lib/Heartbeat.jsm

```diff
@@ -17,9 +19,32 @@ Cu.importGlobalProperties(["URL"]); /* globals URL */
 
 this.EXPORTED_SYMBOLS = ["Heartbeat"];
 
-const log = LogManager.getLogger("heartbeat");
 const PREF_SURVEY_DURATION = "browser.uitour.surveyDuration";
 const NOTIFICATION_TIME = 3000;
+const HEARTBEAT_CSS_URI = Services.io.newURI("resource://shield-recipe-client/skin/shared/Heartbeat.css");
+const HEARTBEAT_CSS_URI_OSX = Services.io.newURI("resource://shield-recipe-client/skin/osx/Heartbeat.css");
+
+const log = LogManager.getLogger("heartbeat");
```

Comment:
This line was moved from above (I re-arranged these to separate CONSTANTS from non-constants). This is used in existing parts of the code, but none of the new code.